specification/metrics/semantic_conventions/http-metrics.md

```diff
@@ -55,6 +55,8 @@ operations. By adding HTTP attributes to metric events it allows for finely tune
 
 ### Metric: `http.server.duration`
 
+**Status**: [Experimental, Feature-freeze][DocumentStatus]
```

Comment:
@trask what does it mean for a metric to have both experimental and Feature freeze simultaneously?

Reply:
Feature-freeze is orthogonal to Experimental/Stable https://github.com/open-telemetry/opentelemetry-specification/blob/ad1537a46eb856eeb915786fd7cf81e07bdeb426/specification/document-status.md?plain=1#L19.